### PR TITLE
BDOC 1.0 distinction from BDOC2.0

### PR DESCRIPTION
**Pull request by sgaktors**
TERA-187
Signed-off-by: Sergei Gluhhov <sergei.gluhhov@aktors.ee>

### CODE REVIEW

**Comment by metsma** [comment]
Why not check mimetype file and its content "application/vnd.bdoc-1.0"? Should be faster than XML parsing

**Comment by sgaktors** [comment]
You are right, I was thinking about mimetype file also.
However, I decided that description of the task was done with a reason.

**Comment by metsma** [comment]
Maybe they didn't know this option exists?

**Comment by sgaktors** [comment]
Finally I have got clarification, that there is no requirement for XML parsing anymore. More effective option will be preferable. I will replace this function with the mimetype check. Thank you for your notes.

File: src/common/Bdoc10Handler.cpp [hunk 1]
```diff
@@ -1,76 +1,39 @@
...
 		    zip_file_t *zipfile = zip_fopen_index(zipConatiner, index, ZIP_FL_UNCHANGED);
 		    if (NULL != zipConatiner) {
-			    const int FILE_CHUNK_SIZE = 1024;
+			    const int FILE_CHUNK_SIZE = 256;
 			    char buf[FILE_CHUNK_SIZE];
-			    QByteArray xmlData;
+			    QByteArray data;
 			    for (;;)
 			    {
 				    zip_int64_t read = zip_fread(zipfile, buf, FILE_CHUNK_SIZE);
```
Comment (metsma) [inline src/common/Bdoc10Handler.cpp]: Why not read only same amount as sizeof(MIMETYPE_BDOC10_APPLICATION).
No need read full file
```
QByteArray data(sizeof(MIMETYPE_BDOC10_APPLICATION), 0);
zip_int64_t read = zip_fread(zipfile, &data[0], data.size());
if(data == MIMETYPE_BDOC10_APPLICATION)
```

**Comment by metsma** [comment]
https://www.kdab.com/goodbye-q_foreach/
and keep tab vs space indenting consistent

**Pull request merged**